Maple2.Server.DebugGame/Graphics/Scene/FollowCameraController.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System.Numerics;
+using Serilog;
+
+namespace Maple2.Server.DebugGame.Graphics.Scene;
+
+public class FollowCameraController : ICameraController {
+    private static readonly ILogger Logger = Log.Logger.ForContext<FollowCameraController>();
+
+    public Camera Camera { get; }
+    public InputState InputState { get; } = new();
+
+
+
+    // Camera follow system
+    public bool IsFollowingPlayer { get; private set; }
+    public long? FollowedPlayerId { get; private set; }
+    public bool HasManuallyStopped { get; private set; }
+
+    // Default camera follow offset
+    private static readonly Vector3 CameraFollowOffset = new Vector3(-800, -1000, 1200);
+
+    // Default camera rotation for MapleStory 2 coordinate system
+    private static readonly Quaternion DefaultCameraRotation = new Quaternion(0.130f, 0.325f, 0.870f, 0.350f);
+
+    // Field overview camera rotation
+    private static readonly Quaternion FieldOverviewRotation = new Quaternion(0.000f, 0.450f, 0.900f, 0.000f);
+
+    public Vector3 CameraTarget { get; private set; } = Vector3.Zero;
+
+    // Camera properties that delegate to Camera.Transform
+    public Vector3 CameraPosition => Camera.Transform.Position;
+    public Quaternion CameraRotation => Camera.Transform.Quaternion;
```

Comment:
Either write to the Transform matrix directly or one of the non-quaternion rotation members. No need to touch quaternions at all for this use case.